cmd/backup/main.go

```diff
@@ -65,4 +82,83 @@ func main() {
 	s.must(s.withLabeledCommands(lifecyclePhaseProcess, s.encryptArchive)())
 	s.must(s.withLabeledCommands(lifecyclePhaseCopy, s.copyArchive)())
 	s.must(s.withLabeledCommands(lifecyclePhasePrune, s.pruneBackups)())
+
+	return nil
+}
+
+func main() {
+	serve := flag.Bool("foreground", false, "run the backup in the foreground")
+	envFolder := flag.String("env-folder", "/etc/dockervolumebackup/conf.d", "location of environment files")
```

Comment:
Not sure if this needs to be configurable. Considering the binary will always be wrapped in a Docker image, we have a pretty clear idea of how the fs layout looks like, so I would think it's ok to hard code it.

Passing it in a flag would also all of a sudden open up a second way of configuring the tool, so if we really want it, it would need to be set in an env var.

I'd vote for keep using the hard coded value.

Reply:
I kept it configurable for ease of testing, but removed from final version.